app/assets/javascripts/discourse/app/lib/page-tracker.js

```diff
@@ -42,8 +42,6 @@ export function startPageTracking(router, appEvents, documentTitle) {
         url,
         title: documentTitle.getTitle(),
         currentRouteName: router.currentRouteName,
-        currentRouteParams: router.currentRoute.params,
-        currentRouteParentName: router.currentRoute.parent?.name,
```

Comment:
These were added and used by Sidebar 6 months ago. I don't think any other thing depends on it so I'm removing without deprecation.